react/react1/week1/todo-list/src/components/TodoList.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import TodoHeader from "./TodoHeader";
+import TodoItem from "./TodoItem";
+
+function TodoList() {
+  return (
+    <div>
+      <TodoHeader />
+      <TodoItem description="Get out of bed" deadline="Wed Sep 13 2017" />
```

Comment:
You definitely did the job but would be interesting to have these items' descriptions and deadlines in objects inside an array that you could map and pass this information as props.

Reply:
Yes, that's a great idea!  Thank you so much for reviewing my hw!